assets/schema_input.json

```
@@ -12,25 +12,22 @@
                 "pattern": "^\\S+$",
                 "errorMessage": "Sample name must be provided and cannot contain spaces"
             },
-            "fastq_1": {
+            "datatype": {
                 "type": "string",
-                "pattern": "^\\S+\\.f(ast)?q\\.gz$",
-                "errorMessage": "FastQ file for reads 1 must be provided, cannot contain spaces and must have extension '.fq.gz' or '.fastq.gz'"
+                "pattern": "^\\S+$",
+                "errorMessage": "Data type, and must be one of: 'pacbio'"
+            },
+            "datafile": {
+                "type": "string",
+                "pattern": "^\\S+$",
```

Any chance a regex like this would work ?
```suggestion
                "pattern": "^\\S+\\.(bam|cram)$",
```

---

updated, plus index file name as well.